### ISSUE TITLE
Initial zenodo mint failed

### ISSUE THREAD

**Issue by glemieux** [NGEET/fates#650]
Looks like the zenodo DOI mint failed due to the `license` field not being recognized.  A google search led me to the following issue on zenodo's page: https://github.com/zenodo/zenodo/issues/1448.  

Based on the comment from zenodo to @ckoven (in a email chain with the CTSM folks), I looked at the REST API documentation (https://developers.zenodo.org/#entities) and found a link to the following for the licensing definitions: https://licenses.opendefinition.org/  

Long story short, I think the license just needs to be `"license":"BSD-3-Clause"`.

FYI, it looks like people have requested a JSON validator as an issue to avoid this problem: https://github.com/zenodo/zenodo/issues/1606

**Comment by glemieux** [comment]
Closed per #651
